R/print.R

```diff
@@ -33,16 +29,47 @@ print_mock_gt <- function(tfrmt, .data = NULL, .default = 1:3, n_cols = 3) {
 #'
 #' @return a stylized gt object
 #' @export
-#' @importFrom gt gt tab_header
+#' @importFrom gt gt tab_header tab_style cell_text cells_body tab_options
+#' @importFrom tidyselect everything
 print_to_gt <- function(tfrmt, .data){
+
   apply_tfrmt(.data, tfrmt, mock = FALSE) %>%
+    cleaned_data_to_gt(tfrmt)
+
+}
+
+
+#' Do all the formatting for the GT
+#'
+#' @param .data cleaned dataset
+#' @param tfrmt tfrmt
+#'
+#' @return GT object
+#' @noRd
+cleaned_data_to_gt <- function(.data, tfrmt){
+  if(is.null(tfrmt$row_grp_style) && length(tfrmt$group) > 0){
```

Comment:
Just pointing out that if someone passes an empty row_group_plan() then we apply the defaults: indenting lower-level groups & output data is grouped by highest level grouping, while here if no row_group_plan() then we do no indentation and group by all vars, is that right?

Reply:
Yeah I put that comment in somewhere. I am not sure if I love that behavior but I was going to try and play around a bit and see how it shakes out. Unless you think we should just change it now. I wanted the indentation to be the default cause that feels like the most common approach but I am might be wrong